cns/restserver/util.go

```diff
@@ -394,9 +394,8 @@ func (service *HTTPRestService) getNetworkContainerResponse(
 		}
 		nmaNCs := map[string]string{}
 		for _, nc := range ncVersionListResp.Containers {
-			nmaNCs[nc.NetworkContainerID] = nc.Version
+			nmaNCs[cns.SwiftPrefix+nc.NetworkContainerID] = nc.Version
```

Comment:
can you fix this in other places for nmaNCs?
I found one more where this bug still exists -
https://github.com/Azure/azure-container-networking/blob/ddb595435169094b3a178f89078e595957ae462a/cns/restserver/util.go#L563

Reply:
spoke with Shufang. Resolving this.